errors/missing-env-value.md

```diff
@@ -0,0 +1,21 @@
+# Missing Env Value
+
+#### Why This Error Occurred
+
+On one of your pages' config you requested an env value that wasn't populated.
+
+```js
+export const config = {
+  env: ['MY_ENV_KEY'],
+}
+```
+
+#### Possible Ways to Fix It
+
+Either remove the requested env value from the page's config or populate it in your `.env` file or manually populate it in your environment before starting `next`
```

Comment:
We need to link to the relevant Next.js documentation.

Reply:
We don't have documentation for this yet, are you mentioning to remind us to add after we do add documentation or do you think we need to add documentation in this PR?

Reply:
I'm good with future docs.